### DISCUSSION TITLE
2-way CalDAV sync

### DISCUSSION THREAD

**Discussion by dawidurbanski** [wpappointments/wpappointments#51] [Ideas]
It should allow for:

- Adding one or many calendars

For each calendar:
- Should block all time that has events in the connected calendar
    - as option
    - enabled by default
- Should add new events to the calendar
   - as option
   - enabled by default
   - adds to first calendar in the list by default
   - allows to pick to which calendars to add events

**Comment by macbaydigital** [comment]
Very important feature! Hope you can bring this on very soon :)

**Comment by wppoland** [comment]
CalDAV sync is on our roadmap. It will ship as a separate free plugin after v1.0 launch. Stay tuned!
